uint16 motor_id
float32 motor_c
